composer.json

```diff
@@ -31,13 +30,15 @@
     "require-dev": {
         "composer/package-versions-deprecated": "^1.11",
         "phpstan/phpstan": "0.12.84",
+        "doctrine/annotations": "^1.0",
         "doctrine/coding-standard": "^6.0 || ^9.0",
         "doctrine/common": "^3.0",
         "phpunit/phpunit": "^7.5.20 || ^8.0 || ^9.0",
         "symfony/cache": "^4.4|^5.0",
         "vimeo/psalm": "4.7.0"
     },
     "conflict": {
+        "doctrine/annotations": "<1.0 || ^2.0",
```

Comment:
if the goal is to allow on 1.x, the second part of the conflict should be `>=2.0`, not `^2.0`. the current conflict rule allows 3.0.